config/src/lib.rs

```diff
@@ -1359,6 +1362,9 @@ impl<'de> Deserialize<'de> for GasLimit {
 pub enum SolidityErrorCode {
```

Comment:
I wonder if there is a way for us to support human-like identifiers for `ignored_error_codes` in our config? Something like (ignores SPDX + some other error code 1234):


```toml
ignored_error_codes = ["SPDX_LICENSE_NOT_PROVIDED", 1234]
```